nmdc_schema/migrators/partials/migrator_from_11_10_0_to_11_11_0/migrator_from_11_10_0_to_11_11_0_part_2.py

```diff
@@ -0,0 +1,51 @@
+from nmdc_schema.migrators.migrator_base import MigratorBase
+
+
+class Migrator(MigratorBase):
+    r"""Migrates a database between two schemas."""
+
+    _from_version = "11.11.0.part_1"
+    _to_version = "11.11.0.part_2"
+
+    def upgrade(self, commit_changes: bool = True) -> None:
+        r"""
+        Migrates the database from conforming to the original schema, to conforming to the new schema. """
+
+        self.adapter.do_for_each_document(
+            "data_object_set", [self.confirm_permissible_values_are_absent]
```

Comment:
The second parameter of `do_for_each_document` is a "callable" rather than a list.

```suggestion
            "data_object_set", [self.confirm_permissible_values_are_absent]
```